setup.py

```diff
@@ -99,6 +99,7 @@
 
 static = False
 debug_flag = False
+fmil_name = "fmilib_shared" if sys.platform.startswith("win") else "fmilib" #If windows shared, else static
```

Comment:
@jschueller I saw your comment here https://github.com/modelon-community/fmi-library/issues/155#issuecomment-2833547365 as well. I wonder if we should just change the build system to always use static (or shared) for both platforms. I think the reason why this exists currently in the code is an old legacy reason where we had problems creating a static lib on Windows.

Reply:
yes, lets use fmilib_shared by default everywhere
I think we should keep this option for retrocompatibility in case we change fmillib default lib name
(I have in mind that we should avoid creating both shared and static lib in fmilib but allow either one with the same library name, ie this might change the default name to just "fmilib")